infra/resources/modules/commons/containers.tf

```diff
@@ -45,3 +45,65 @@ resource "azurerm_cosmosdb_sql_container" "subscription_history" {
     }
   }
 }
+
+resource "azurerm_cosmosdb_sql_container" "activations" {
+  name                  = "activations"
+  resource_group_name   = azurerm_resource_group.data_rg.name
+  account_name          = module.cosmosdb_account.name
+  database_name         = module.cosmosdb_sql_database_trial.name
+  partition_key_path    = "/trialId"
+  partition_key_version = 2
+
+  autoscale_settings {
+    max_throughput = 1000
+  }
+
+  indexing_policy {
+    included_path {
+      path = "/*"
+    }
+
+    excluded_path {
+      path = "/\"_etag\"/?"
+    }
+
+    composite_index {
+      index {
+        path  = "/trialId"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/type"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/isActive"
```

Comment:
**suggestion**: This boolean indicates that a specific participation request has been marked as activated; it does not express the activation status of the request. In fact, the request to participate in the trial might not be active even if this value is true (e.g. because it has been disabled). For this reason, I believe the name `activated` is more suitable, as it represents that it has been activated at some point, while the name `isActive` denotes that the request is currently active, which could be not true. What do you think?

```suggestion
        path  = "/activated"
```